src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2708,21 +2713,27 @@ module Types = struct
       type input = SendTestZkappInput.input
 
       let arg_typ =
-        let conv (x : Parties.t Fields_derivers_graphql.Schema.Arg.arg_typ) :
-            Parties.t Graphql_async.Schema.Arg.arg_typ =
+        let conv
+            (x :
+              Mina_base.Zkapp_command.t
+              Fields_derivers_graphql.Schema.Arg.arg_typ ) :
+            Mina_base.Zkapp_command.t Graphql_async.Schema.Arg.arg_typ =
           Obj.magic x
         in
         let arg_typ =
-          { arg_typ = Parties.arg_typ () |> conv
+          { arg_typ = Mina_base.Zkapp_command.arg_typ () |> conv
           ; to_json =
-              (function x -> Yojson.Safe.to_basic (Parties.parties_to_json x))
+              (function
+              | x ->
+                  Yojson.Safe.to_basic
+                    (Mina_base.Zkapp_command.zkapp_command_to_json x) )
           }
         in
         obj "SendZkappInput" ~coerce:Fn.id
           ~split:(fun f (x : input) -> f x)
           ~fields:
-            [ arg "parties"
-                ~doc:"Parties structure representing the transaction"
+            [ arg "zkapp_command"
+                ~doc:"Zkapp_command structure representing the transaction"
```

Comment:
Am I right that the graphql field should be spelled "zkappCommand"?
Also, the doc string shouldn't have an underscore: `"Zkapp command structure representing the transaction"`